Styling/README.md

```diff
@@ -621,7 +621,9 @@ The prefix `tiles3d_` is reserved for built-in variables. The following built-in
 
 #### tiles3d_tileset_time
 
-Gets the time, in seconds, since the tileset was first loaded. This is useful for creating dynamic styles that change with time.
+Gets the time, in milliseconds, since the tileset was first loaded. This is useful for creating dynamic styles that change with time.
+
+_Implementation note: If this value is stored as a 32-bit floating point number, be aware of increasing loss of precision over time._
```

Comment:
It's needs some context.  As is, readers will see it as too scary to use or just ignore the warning.